config/locales/emails/candidate_mailer.yml

```diff
@@ -92,12 +92,12 @@ en:
         study_mode_full: "%{study_mode} places for %{course_name} at %{provider_name} became full while %{candidate_name} was awaiting references"
     ucas_match_initial_email:
       duplicate_applications:
-        subject: 'Action required: it looks like you submitted a duplicate application'
+        subject: "Withdraw your duplicate application by %{withdraw_by_date}"
       multiple_acceptances:
         subject: 'Action required: it looks like you’ve accepted two offers'
     ucas_match_reminder_email:
       duplicate_applications:
-        subject: 'Action required: please withdraw one of your duplicate applications'
+        subject: "Withdraw your duplicate application by %{withdraw_by_date}"
```

Comment:
Consider refactoring duplicated subject so we can reuse the same reference for both Ucas emails.